### ISSUE TITLE
Missing newline after `namedParameter="""string"""`

### ISSUE THREAD

**Issue by ZakTaccardi** [pinterest/ktlint#898]
## Expected Behavior
I expect the following to pass formatting.

```kotlin
json(jsonString = """{ "id": "1234" }""")
```

## Observed Behavior
Fails with: `Missing newline after "=" (indent)`

## Steps to Reproduce
```kotlin
fun json(jsonString: String?) {
    TODO()
}

fun exampleThatFailsKtLint() {
    json(jsonString = """{ "id": "1234" }""")
}

fun exampleThatPassesKtLint() {
    // Android Studio formatting will move this back to next example, which will fail ktlint
    json(
        jsonString =
            """{ "id": "1234" }"""
    )
}

fun exampleThatFailsKtLintWhichAndroidStudioAutoFormatsTo() {
    json(
        jsonString =
        """{ "id": "1234" }"""
    )
}

fun exampleThatPassesLint() {
    // lacks named parameter
    json(
        """{ "id": "1234" }"""
    )
}
fun exampleThatPassesLint2() {
    // lacks named parameter
    json("""{ "id": "1234" }""")
}
```

## Your Environment
* Version of ktlint used: `0.38.1` (command line)
* AS `4.0.1` with AGP `3.6`
* Operating System and version: MacOS 10.15.6

**Comment by ZakTaccardi** [comment]
I also tried disabling this but it did not seem to work?

```kotlin
/* ktlint-disable indent */
json(jsonString = """{ "taskReferenceId": "1234" }""")
/* ktlint-enable indent */
```

**Comment by Tapchicoma** [comment]
I could not reproduce your issue with build from current `master`.  Most probably it was fixed via #711 issue.

Please open a separate issue regarding "indent" rule disabling is not working.

**Comment by romtsn** [comment]
Yeah this is fixed already on master. Disabling the indent rule is also not possible at the moment as it runs all or nothing, gonna be possible after we split it as mentioned in #764
